scripts/cli/install.sh

```diff
@@ -6,11 +6,30 @@ USER_BIN_DIR="$HOME/bin"
 BASE_DIR="$HOME/.agc"
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 
+selectArch () {
+    if [[ $(arch) == "arm64" ]]; then
+        eval $1="arm64"
+    else
+        if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+            eval $1="amd64"
+        fi
+    fi
+}
+
 selectCliFile () {
+    selectArch archKind
+    fileName="agc"
+
+    if [[ -z "$archKind" ]]; then
+        eval $filename="agc"
+    else
+        eval $fileName="$filename-$archKind"
+    fi
+
     if [[ "$OSTYPE" == "linux-gnu"* ]]; then
-        eval $1="agc-amd64"
+        eval $1=filename
```

Comment:
I think this change is breaking the script.

Reply:
@mattcarp12 what are you seeing here? Is there an error that's getting output? Is the script exiting? Is the binary being misnamed?

Reply:
Sorry, yes when I run this on linux I get `cp: cannot stat ‘/home/carpedma/amazon-genomics-cli/dist/amazon-genomics-cli/filename’: No such file or directory`

Reply:
This is definitely a misnamed variable. I'm fixing that, but also making sure nothing else problematic is happening.

Reply:
Ok, fix is in a PR since this one is closed: https://github.com/aws/amazon-genomics-cli/pull/575